model.ipynb

```diff
@@ -33,7 +33,7 @@
    "cell_type": "markdown",
```

Comment:
Good job keeping the model and tutorial files up to date! At some point we might want to decide if it is worth keeping both of these around and maintaining them. Maybe they can be rolled into a single file that explains how to use the code and also the underlying model.